<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="s5inc">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s4inc">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s7ON">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s3inc">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s1inc">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s4ON">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s1OFF">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s8inc">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s2ON">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s6dec">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s1dec">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s3dec">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s4OFF">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s2inc">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savemessage">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s6ON">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s7dec">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s4dec">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s5OFF">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s2OFF">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s7OFF">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendmessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s6OFF">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s5ON">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s7inc">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s2dec">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s8dec">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s8OFF">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s6inc">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s8ON">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s3ON">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s3OFF">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s1ON">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s5dec">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>270</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="s8">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="s4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="s6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="s2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="s5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="s1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="s3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="s7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setmessage">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Message</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brute">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Brute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="idinc">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iddec">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCAN_RE_Tool">
    <property name="title">
     <string>CAN RE Tool</string>
    </property>
   </widget>
   <addaction name="menuCAN_RE_Tool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
